package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
Ah, ok. Right I see the issue you're having. I've created https://github.com/gaearon/babel-plugin-react-transform/issues/35 as this is a babel-plugin-react-transform issue.